src/index.js

```diff
@@ -50,20 +54,55 @@ const hunt = () => {
   return enemy.msgs.encounter;
 };
 
+/**
+ * @param {number} low The lower bound (inclusive)
+ * @param {number} high The upper bound (exclusive)
+ * @returns {number} A random number between low and high
+ */
```

Comment:
Did they show you [JSDoc](https://jsdoc.app/)? This is the "standard" (not officially, but accepted) way of formally documenting JS functions. You don't have to follow this always, regular comments are fine, but I write them this way to make things clear. 

It should probably highlight in your editor which is nice, might even give you some intelligence when you hover over stuff if you got that much set up? Dunno how set up yours is but we can try stuff out one day it makes it easier to see everything if done well.

Reply:
They did not! I'll follow your lead with this standard, since it looks better than my comments thus far!